railties/lib/rails/generators/rails/authentication/templates/mailers/passwords_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class PasswordsMailer < ApplicationMailer
+  def reset(user)
+    @token = user.signed_id(purpose: "password", expires_in: 15.minutes)
```

Comment:
Hmm, I wonder if we should simply bake this into has_secure_password. That it automatically sets this up with the token. Because I'm finding that adding this into the model seems overly heavy. Especially since it only barely encapsulates one line from the mailer. Not sure the abstraction carries its own weight with that alone.

Reply:
That makes a lot of sense to me!

Reply:
Actually I'm going to do that. Opening another PR for that shortly.

Reply:
Added https://github.com/rails/rails/pull/52483